includes/multi-currency/MultiCurrency.php

```diff
@@ -240,6 +247,8 @@ public function __construct( WC_Payments_API_Client $payments_api_client, WC_Pay
 			add_action( 'init', [ $this, 'possible_simulation_activation' ], 13 );
 			add_action( 'woocommerce_created_customer', [ $this, 'set_new_customer_currency_meta' ] );
 		}
+
+		$this->currency_switcher_block->init_hooks();
```

Comment:
I was unsure of how to include the `init_hooks` call for the `currency_switcher_block` due to it is created in the constructor of this class. Including here causes no side effects.